### PR TITLE
Create template for PRs in .github directory #7

### LINKED ISSUE #8: Create template for new issues description in .github
## Description
We want to have a template that automatically appears when creating a new issue. This can be done by adding a proper-named MD file to the .github directory.
  - Create a new directory inside .github called `ISSUE_TEMPLATE`
  - Create a proper template using markdown that includes a base to be filled by those who propose an issue.

### PR DESCRIPTION
**Pull request by MariangelaNM**
# 📝 Create template for PRs in .github directory #7

## 🛠️ Issue
- Closes #8 

## 📖 Description
This pull request introduces a standardized template for all future pull requests in the repository. The template aims to improve the clarity and consistency of PR submissions, making it easier for reviewers to understand the changes proposed and the reasoning behind them.

## 🔍 Changes Made
- Created a new directory `.github/PULL_REQUEST_TEMPLATE`.
- Added a markdown file `template.md` for the PR template.
- Included sections for description, checklist, changes made, related issues, screenshots, and additional notes.

## 📸 Screenshots (if applicable)
-

## 📝 Additional Notes
- Add any relevant note that can enrich the description of the PR.

### CODE REVIEW

**Comment by MariangelaNM** [comment]
@danielcdz

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## 📖 Description
+Please provide a clear and concise description of your changes. Include any relevant background information or context that may help reviewers understand the purpose of this PR.
+
+### ❓ What does this PR do?
+- Briefly describe the functionality or feature that this PR introduces or modifies.
+
+### 🚀 Why is this change necessary?
+- Explain the rationale behind this change. What problem does it solve, or what benefit does it provide?
+
+## ✅ Checklist
```
Comment (danielcdz) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:12]: This section can be avoided/deleted

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# 📝 Pull Request Title
+
+## 📖 Description
+Please provide a clear and concise description of your changes. Include any relevant background information or context that may help reviewers understand the purpose of this PR.
+
+### ❓ What does this PR do?
+- Briefly describe the functionality or feature that this PR introduces or modifies.
+
+### 🚀 Why is this change necessary?
```
Comment (danielcdz) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:9]: This section can be deleted also

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# 📝 Pull Request Title
+
+## 📖 Description
+Please provide a clear and concise description of your changes. Include any relevant background information or context that may help reviewers understand the purpose of this PR.
```
Comment (danielcdz) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:4]: Change this to: 
Please provide a general description of your changes. Include any relevant background information or context that may help reviewers understand the purpose of this PR.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+## 🔍 Changes Made
+List the specific changes made in this PR. Use bullet points for clarity:
+- 
+- 
+- 
+
+## 🔗 Related Issues
+Link any related issues or tickets this PR addresses:
+- Closes #ISSUE_NUMBER
+- Related to #ISSUE_NUMBER
```
Comment (danielcdz) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:30]: This can be deleted

**Review by danielcdz — CHANGES_REQUESTED** [review]

**Comment by danielcdz** [comment]
@MariangelaNM Thank you! I left some comments

**Comment by danielcdz** [comment]
@MariangelaNM can you left the template with the sections I added to the description of this PR?

**Comment by MariangelaNM** [comment]
# 📝 Create template for PRs in .github directory #7

## 📖 Description
This pull request introduces a standardized template for all future pull requests in the repository. The template aims to improve the clarity and consistency of PR submissions, making it easier for reviewers to understand the changes proposed and the reasoning behind them.

## 📸 Screenshots (if applicable)
-

## 📝 Additional Notes
- Add any relevant note that can enrich the description of the PR.

**Comment by MariangelaNM** [comment]
@danielcdz  See how the implementation would be

**Review by danielcdz — COMMENTED** [review]
LGTM

**Review by danielcdz — APPROVED** [review]

**Pull request merged**